KIPs/kip-82.md

```diff
@@ -0,0 +1,334 @@
+- Associate GC reward with the growth of the Klaytn ecosystem
+
+## Specification
+Block generation is rewarded for the node operations that are serving as public goods in the Klaytn network. CN and PN are in charge of block creation and propagation while EN is in charge of blockchain data inquiry based on an individual's needs. The infrastructure cost remains similar, enabling some fixed benefits. Therefore, fixed compensation will be equivalent to the sum of the transaction fee and an additional block compensation, which can be both inflationary and deflationary.
+
+The block proposal method will be based on equal selection regardless of the staking amount. The current model provides a block generation reward based on staking amount and the Gini-coefficient. Until now, the Gini coefficient was utilized to alleviate the chance of an entity with more staking amount being selected as a block proposer. As this model provided a fairly equal chance to every participant, each node was promised a certain amount of rewards.
+
+In the new method, the reward for node operation is required since the staking amount is not considered in the proposal selection process. Therefore, 20 percent of the total reward amount will be placed as a node operation reward.
+
+Node operators will also be compensated with the staking for enhancing network stability from an economic standpoint. The opportunity cost incurred from staking is rewarded based on inflation. In this system, inflation provides a tax-like common effect. Public users can also participate and receive rewards through the GC-run public staking program.
+
+Of the 300 million KLAYs that are minted annually, only 100 million go to GC, and 20% and 80% of the distribution is from this 100 million. The renewal separates GC reward into two groups: basic reward and staked reward. The basic reward is compensating for node operation by providing 20% of GC reward and 100% of gas fee resources from transactions. The staked reward provides only the rest of 80% of the minting resource based on the total stake amount.
+
+![gc reward allocation](../assets/kip-82/gc_reward_allocation.png)
+
+After the Magma hard fork, Klaytn has been burning the first half of the gas fee based on the KIP-71 proposal while the second half is distributed. The remaining second half is distributed into GC reward, Klaytn Improvement Reserve (KIR) and Klaytn Growth Fund (KGF).
+
+The renewal will include a change of gas fee as follows.
+
+In addition to changing the reward structure, this renewal will include burning the gas fee that is provided as GC’s basic reward. The KGF and KIR portion will be excluded from the gas fee.
+If the amount of gas fee that is provided as a basic reward does not exceed a certain level, all of them will be burned. The threshold amount is 20% of minting provided as a basic reward. If the gas fee exceeds this amount, 20% will be incinerated and the additional amount will be distributed to GC.
```

Comment:
@yeri-lee Since the pseudocode below uses 'burnt', it would be good to use the same past participle here as well.

Reply:
@smseo I have revised.